File: day_15.py
def get_present_sheep(array):
    present = 0
    for sheep in array:
        if sheep == True:
            present += 1
    return present


sheep_array = [True,  True,  True,  False,  True,  True,  True,  True ,True,  False, True, False,  True,  False, False, True ,True,  True,  True,  True ,False, False, True,  True,]
result = get_present_sheep(sheep_array)
print("Number of sheep present: ",result)

File: day_15.js
function get_present_sheep(array){
    let present = 0;
    for (let sheep of array){
        if (sheep == true){
            present++;
        }
    }
    return present;
}

sheep_array = [true,  true,  true,  false,  true,  true,  true,  true ,true,  true, true, true,  true,  false, false, true ,true,  true,  true,  true ,false, false, true,  true,]
result = get_present_sheep(sheep_array)
console.log("number of present sheeps: ",result)